models/public/aclnet-int8/labels.txt

```diff
@@ -0,0 +1,53 @@
+Dog
```

Comment:
I think labels should be places in the same directory with other labels (data directory)

Reply:
agree, moved to data subfolder